connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -27,11 +27,13 @@
 import org.apache.kafka.connect.errors.NotFoundException;
 import org.apache.kafka.connect.runtime.AbstractHerder;
 import org.apache.kafka.connect.runtime.ConnectorConfig;
+import org.apache.kafka.connect.runtime.ConnectorStatus;
```

Comment:
@kkonstantine agreed about not using magic params, what do you think about the suggestion in https://github.com/apache/kafka/pull/1788#discussion_r76697445? seems like the cleanest solution to me.